dist/10-10EZ-schema.json

```diff
@@ -272,18 +272,18 @@
       "properties": {
         "first": {
           "type": "string",
-          "pattern": "^.*\\S.*",
           "minLength": 1,
-          "maxLength": 30
+          "maxLength": 30,
+          "pattern": "^.*\\S.*"
```

Comment:
Does this order matter?

Reply:
order of object keys?

Reply:
Yeah was just wondering if this was changed on purpose or just some undone edits.

Reply:
the order shouldn't matter, order of object keys isn't guaranteed in js